sonar-project.properties

```diff
@@ -1,11 +1,16 @@
+sonar.organization=camel-tooling
 sonar.projectKey=camel-lsp-client-vscode
 sonar.projectName=VS Code client for Apache Camel Language Server
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/ 
 sonar.sources=src,scripts
 sonar.exclusions=src/test/**,src/ui-test/**
-sonar.tests=src/test,src/ui-test
-sonar.organization=camel-tooling
+sonar.tests=src/test
 
 sonar.links.homepage=https://github.com/camel-tooling/camel-lsp-client-vscode
 sonar.links.ci=https://app.circleci.com/pipelines/github/camel-tooling/camel-lsp-client-vscode
 sonar.links.scm=https://github.com/camel-tooling/camel-lsp-client-vscode
 sonar.links.issue=https://github.com/camel-tooling/camel-lsp-client-vscode/issues
+
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
```

Comment:
better if extra end of line is added at the end of files. it allows better diff on next modifications of the files